receiver/sqlqueryreceiver/metrics.go

```diff
@@ -27,6 +29,8 @@ func rowToMetric(row metricRow, cfg MetricCfg, dest pmetric.Metric) error {
 	dest.SetUnit(cfg.Unit)
 	dataPointSlice := setMetricFields(cfg, dest)
 	dataPoint := dataPointSlice.AppendEmpty()
+	dataPoint.SetStartTimestamp(pcommon.NewTimestampFromTime(time.Now()))
```

Comment:
start time cannot be set to the same timestamp. If has different meaning depending on the data type.
- Gauge: shouldn't have a start time.
- Cumulative sum should have a start time set to the beginning of the data points cumulation,
- Non-cumulative sum should have a start time set to the previous endpoint

Check https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/data-model.md#metric-points for more details

Reply:
Unless the receiver maintains the state of previous metrics how would one get the previous points timestamp OR the timestamp of the original metric? @dmitryax  

@pmcollins Any recommendations ?

Reply:
Just `current time - scrape duration` should be sufficient for non-cumulative sum. And the receiver start time should be ok for cumulative sum.

Not sure why do we need sums at all to be honest. Gauge seems more appropriate for this kind of random data